config.js

```diff
@@ -389,11 +386,11 @@ if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxranke
 
     if (argv.banunranked) {
         for (let i of argv.banunranked.split(',')) {
```

Comment:
Consider a different variable name here. `i` is not very informative, and usually used for integers, here it's a string, or actually a username.

Reply:
i used the same as was here

indeed, `i` is used for counters :+1: , even i know that :rofl: 

out of curiosity, what name should be used in that case ?